tests/test_vector2_ctor.py

```diff
@@ -38,3 +40,9 @@ def test_ctor_noncopy_subclass():
 def test_ctor_noncopy_superclass():
     v = Vector2(1, 2)
     assert V(v) is not v
+
+
+@given(v=vectors())
+def test_ctor_pickle(v: Vector2):
+    """Test that Vector2 instances can be pickled."""
```

Comment:
```suggestion
    """Round-trip vectors through `pickle.{dumps,loads}`."""
```

We aren't just testing that they can be pickled, but that unpickling gives the correct result.